packages/node/bench/package.json

```diff
@@ -1,11 +0,0 @@
-{
-  "private": true,
-  "name": "bench",
-  "version": "1.0.0",
-  "main": "index.js",
-  "license": "UNLICENSED",
-  "dependencies": {
-    "express": "4.17.3",
-    "fs-extra": "8.0.1"
-  }
-}
```

Comment:
This was left over from https://github.com/vercel/vercel/pull/12697

We could remove it in a separate PR, but CI is being touchy enough as it is.